kotest-assertions/kotest-assertions-core/src/commonMain/kotlin/io/kotest/matchers/ranges/Range.kt

```diff
@@ -25,8 +25,8 @@ internal data class Range<T: Comparable<T>>(
       val endOfThis: T = this.end.value
       val startOfOther: T = other.start.value
       return when {
-         (this.end.edgeType== RangeEdgeType.INCLUSIVE && other.start.edgeType == RangeEdgeType.INCLUSIVE) -> (endOfThis < startOfOther)
-         else -> (endOfThis <= startOfOther)
+         (this.end.edgeType == RangeEdgeType.INCLUSIVE && other.start.edgeType == RangeEdgeType.INCLUSIVE) -> (endOfThis < startOfOther)
+         else                                                                                              -> (endOfThis <= startOfOther)
```

Comment:
What formatting rule is this? I don't think it's one we typically apply in this project :)

Reply:
Oh oops, I must have used my default formatter! I enabled 'align when branches in columns'.

Thanks for checking, I'll fix my config.